### DISCUSSION TITLE
Can Not Install Extension Pg_partman

### DISCUSSION THREAD

**Discussion by polenvinagre** [supabase/supabase#14506] [Feature Requests]
# Bug report

- [ ] I confirm this is a bug with Supabase, not with my own application. 
- [ ] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

I can not install Extension Pg_partam.
https://github.com/pgpartman/pg_partman

It already supported by Aws Postgres: 
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/PostgreSQL_Partitions.html

## To Reproduce

Just follow the steps to install extension as documented and try install pg_partaman

## Expected behavior

Install and use extension Pg_partman

## System information

- OS: [e.g. macOS, Windows] Windows 10
- Browser (if applies) [e.g. chrome, safari]: Chrome 
- Version of supabase-js: [e.g. 6.0.2] actual
- Version of Node.js: [e.g. 10.10.0]

## Additional context

What I need to do to install extension pg_partman.
If I do not have permission. What is necessary to Supabase team install it for me?
Thanks in advance.

**Comment by GaryAustin1** [comment]
You cannot install extensions on hosted projects unless they are entirely SQL based.

There is a thread for requesting new extensions, that one is not in it.
https://github.com/orgs/supabase/discussions/679

I do not know if Supabase will install an extension for you on a Pro Plan.  You would need to contact support.  

Moving this to discussion as more a feature request and not a bug.

**Comment by polenvinagre** [comment]
Thanks for the feedback.

**Comment by evelant** [comment]
This is confusing because the [Supabase docs mention pg_partman](https://supabase.com/docs/guides/database/partitions#other-tools) as a useful tool to manage partitions but Supabase doesn't include pg_partman in the available extensions.

**Reply by Kilowhisky** [reply]
It used to be there. I used to use it and ran into huge issues with migrations and pushing/pulling from local dbs and had to bail on it. 

Looking back now, it appears to be gone.
